test/gtest/solver_bwd.hpp

```diff
@@ -28,32 +28,46 @@
 #include <gtest/gtest.h>
 #include "conv_common.hpp"
 #include "get_handle.hpp"
-#include "tensor_util.hpp"
-#include <fusionHost.hpp>
 #include <miopen/conv/data_invoke_params.hpp>
-
-#include <hip_float8.hpp>
-#include <miopen/type_name.hpp>
-#include <miopen/rank.hpp>
+#include <miopen/solver.hpp>
 
 #include "conv_test_base.hpp"
 #include "conv_tensor_gen.hpp"
 
-#include "get_solver.hpp"
 #include "../workspace.hpp"
 
 template <typename T = float, typename Tref = float, bool use_cpu_ref = false>
 struct ConvBwdSolverTest
     : public ::testing::TestWithParam<std::tuple<miopenConvFwdAlgorithm_t, ConvTestCaseBase>>
 {
+    void SolverBwd(const miopen::solver::conv::ConvSolverBase& solv)
+    {
+        // SetUpTest() and TearDownTest() are moved here so that if a test is skipped, time is not
+        // wasted.
```

Comment:
...But actually I highly recommend keeping the functionality of `SetUp` and `TeadDown`, BUT skipping the test right in SetUp, see http://google.github.io/googletest/reference/testing.html#GTEST_SKIP